app/src/main/res/values/styles.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <style name="ButtonStyle">
+        <item name="android:layout_width">@dimen/width_button_xbig_60</item>
```

Comment:
as a standard (and good practice) the **android:layout_width** and the **android:layout_height** should be in the XML, not in a style

Reply:
Added layout_width and layout_height in activity_main and removed from styles.